index.html

```diff
@@ -0,0 +1,256 @@
+            <a class="english-link" href="#english">
+              English
+            </a>
+          </li>
+          <li class="item">
+            <a class="myPage-link" href="#myPage">
+              My Page
+            </a>
+          </li>
+          <li class="item">
+            <a class="login-link" href="#login">
+              Login
+            </a>
+          </li>
+        </nav>
+      </div>
+      <div class="mainNav">
+        <a href="#home">
+          <figure class="logoBox">
+            <svg viewBox="0 0 100 100" class="circle-waveform-lines">
+              <use  xlink:href="sprite.svg#circle-waveform-lines"></use>
+            </svg> 
+            <div class="logoText">
+              <h3>Regga<span class="red"><em>educa</em></span>ton</h3>
+              <h5>Concert Tour 2023</h5>
+            </div>
+          </figure>
+        </a>
+        <nav class="main">
+          <li class="item">
+            <a class="about-link" href="#about">
+              About
+            </a>
+          </li>
+          <li class="item">
+            <a class="join-link" href="#join">
+              Join
+            </a>
+          </li>
+          <li class="item">
+            <a class="sponsor-link" href="#sponsor">
+              Sponsor
+            </a>
+          </li>
+          <li class="item">
+            <a class="news-link" href="#news">
+              News
+            </a>
+          </li>
+        </nav>
+        <button type="button" class="getTicket">Purchase Tickets</button>
+      </div>
+      <div class="mobileNav">
+        <svg viewBox="0 0 100 100" class="menu-burger">
+          <use  xlink:href="sprite.svg#menu-burger"></use>
+        </svg>
+      </div>
+    </header>
+    <div id="page" class="homePage">
+      <section class="hero">  
+        <article class="card">
+          <h3>"Say It Loud!"</h3>
+          <h1>
+            <span class="line1">
+            Regga<em>educa</em>ton
+            </span>
+            <br class="about">
+            <span class="line2">Concert Tour</span>
+            <span class="line3"> 2023</span>
+            <svg viewBox="0 0 100 100" class="microphone-alt">
+              <use xlink:href="sprite.svg#microphone-alt"></use>
+            </svg>
+          </h1>
+          <p>
+            <span class="home">Get ready to vibe and make a real impact with the hottest reggaeton tour in town, putting your dance moves to work for our local schools. Let's hit the floor, and turn up the education!</span>
+
+            <span class="about">Groove to the infectious tunes while knowing that your ticket purchase is helping to pave the way for a brighter future. Proceeds from this unforgettable concert tour will directly benefit our community's education initiatives, providing vital resources, scholarships, and opportunities for students of all ages. Every move you make on the dance floor contributes to building a stronger foundation for learning and growth.</span>
```

Comment:
In the mobile version of your application, I suggest that you increase the size of the text. This will help people with poor sight to easily interact with the content of your webpage and also ensure that we match the original design as much as possible.
 
 Your design 😎 
![image](https://github.com/Drod86/reggaeducaton/assets/99394374/82a26c0e-c7d7-4e58-b5cd-044779d62a6b)

Original design 😎 
![image](https://github.com/Drod86/reggaeducaton/assets/99394374/bd9eacd8-bbd7-4ec4-ab91-8c8be39522f9)